specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -807,35 +807,35 @@ for Containers specification.
 === Platform Contracts
 
 This section describes the Java Platform,
-Enterprise Edition contracts that must be fulfilled by a Java EE Product
-Provider implementing the full Java EE platform. Java EE profiles may
+Enterprise Edition contracts that must be fulfilled by a Jakarta EE Product
+Provider implementing the full Jakarta EE platform. Jakarta EE profiles may
 include some or all of these facilities, as described in
 link:#a3212[See Profiles]”.
 
-=== Java EE APIs
+=== Jakarta EE APIs
 
-The Java EE APIs define the contract between the
-Java EE application components and the Java EE platform. The contract
+The Jakarta EE APIs define the contract between the
+Jakarta EE application components and the Java EE platform. The contract
 specifies both the runtime and deployment interfaces.
 
-The Java EE Product Provider must implement the
-Java EE APIs in a way that supports the semantics and policies described
+The Jakarta EE Product Provider must implement the
+Jakarta EE APIs in a way that supports the semantics and policies described
 in this specification. The Application Component Provider provides
 components that conform to these APIs and policies.
 
-=== Java EE Service Provider Interfaces (SPIs)
+=== Jakarta EE Service Provider Interfaces (SPIs)
 
-The Java EE Service Provider Interfaces (SPIs)
-define the contract between the Java EE platform and service providers
-that may be plugged into a Java EE product. The Connector APIs define
+The Jakarta EE Service Provider Interfaces (SPIs)
+define the contract between the Jakarta EE platform and service providers
+that may be plugged into a Jakarta EE product. The connector APIs define
 service provider interfaces for integrating resource adapters with a
-Java EE application server. Resource adapter components implementing the
-Connector APIs are called Connectors. The Java EE Authorization APIs
+Jakarta EE application server. Resource adapter components implementing the
+connector APIs are called Connectors. The Jakarta EE authorization APIs
```

Comment:
Capitalize Authorization

Reply:
It's just "Jakarta Authorization", no "EE".